cnf-tests/Dockerfile

```diff
@@ -54,20 +56,28 @@ RUN yum install -y numactl-devel make gcc && \
     curl -O $RT_TESTS_URL/$RT_TESTS_PKG.tar.gz && \
     tar -xvf $RT_TESTS_PKG.tar.gz && \
     cd $RT_TESTS_PKG && \
-    make oslat && \
-    cp oslat /oslat
+    make oslat hwlatdetect cyclictest && \
+    cp oslat /oslat && \
+    cp hwlatdetect /hwlatdetect && \
+    cp cyclictest /cyclictest
 
 FROM centos:7
 
-RUN yum install -y lksctp-tools iproute libhugetlbfs-utils libhugetlbfs tmux ethtool ping numactl-libs linuxptp iperf3
+RUN yum install -y lksctp-tools iproute libhugetlbfs-utils libhugetlbfs tmux ethtool ping numactl-libs linuxptp iperf3 python3
```

Comment:
I take it that hw latency can only run with python3 ?

Reply:
+1 can we clarify why this is needed now?

Reply:
I didn't check with python2, but yes we need python for that

Reply:
ok, and which package requires python? just a comment here is enough.